frontend/ui/data/dictionaries.lua

```diff
@@ -1195,6 +1195,2750 @@ local dictionaries = {
+    },
+    {
+        name = "Plautdietsch-English Wiktionary dictionary",
+        lang_in = "Plautdietsch",
+        lang_out = "English",
+        entries = 2993,
+        license = "Dual-licensed under CC-BY-SA 3.0 and GFDL",
+        url = "https://github.com/Vuizur/Wiktionary-Dictionaries/raw/master/Plautdietsch-English%20Wiktionary%20dictionary%20stardict.tar.gz",
+    },
+    {
+        name = "Polish-English Wiktionary dictionary",
+        lang_in = "Polish",
+        lang_out = "English",
+        entries = 51071,
+        license = "Dual-licensed under CC-BY-SA 3.0 and GFDL",
+        url = "https://github.com/Vuizur/Wiktionary-Dictionaries/raw/master/Polish-English%20Wiktionary%20dictionary%20stardict.tar.gz",
+    },
+    {
+        name = "Portuguese-English Wiktionary dictionary",
+        lang_in = "Portuguese",
+        lang_out = "English",
+        entries = 52502,
+        license = "Dual-licensed under CC-BY-SA 3.0 and GFDL",
+        url = "https://github.com/Vuizur/Wiktionary-Dictionaries/raw/master/Portuguese-English%20Wiktionary%20dictionary%20stardict.tar.gz",
+    },
+    {
+        name = "Prakrit-English Wiktionary dictionary",
+        lang_in = "Prakrit",
+        lang_out = "English",
+        entries = 912,
+        license = "Dual-licensed under CC-BY-SA 3.0 and GFDL",
+        url = "https://github.com/Vuizur/Wiktionary-Dictionaries/raw/master/Prakrit-English%20Wiktionary%20dictionary%20stardict.tar.gz",
+    },
+    {
+        name = "Proto-Brythonic-English Wiktionary dictionary",
+        lang_in = "Proto-Brythonic",
+        lang_out = "English",
+        entries = 587,
+        license = "Dual-licensed under CC-BY-SA 3.0 and GFDL",
+        url = "https://github.com/Vuizur/Wiktionary-Dictionaries/raw/master/Proto-Brythonic-English%20Wiktionary%20dictionary%20stardict.tar.gz",
+    },
+    {
+        name = "Proto-Celtic-English Wiktionary dictionary",
+        lang_in = "Proto-Celtic",
+        lang_out = "English",
+        entries = 782,
+        license = "Dual-licensed under CC-BY-SA 3.0 and GFDL",
+        url = "https://github.com/Vuizur/Wiktionary-Dictionaries/raw/master/Proto-Celtic-English%20Wiktionary%20dictionary%20stardict.tar.gz",
+    },
+    {
+        name = "Proto-Finnic-English Wiktionary dictionary",
+        lang_in = "Proto-Finnic",
+        lang_out = "English",
+        entries = 1622,
+        license = "Dual-licensed under CC-BY-SA 3.0 and GFDL",
+        url = "https://github.com/Vuizur/Wiktionary-Dictionaries/raw/master/Proto-Finnic-English%20Wiktionary%20dictionary%20stardict.tar.gz",
+    },
+    {
+        name = "Proto-Germanic-English Wiktionary dictionary",
+        lang_in = "Proto-Germanic",
+        lang_out = "English",
+        entries = 4798,
+        license = "Dual-licensed under CC-BY-SA 3.0 and GFDL",
+        url = "https://github.com/Vuizur/Wiktionary-Dictionaries/raw/master/Proto-Germanic-English%20Wiktionary%20dictionary%20stardict.tar.gz",
+    },
```

Comment:
I use these types of entries regularly actually, though perhaps not on my Kobo. I tend to prefer to use a second device or a paper dictionary unless I'm on the train or something.